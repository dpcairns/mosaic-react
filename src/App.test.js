// import React from 'react';
// import renderer from "react-test-renderer";

// // import App from './App';
// import Landing from './Landing.js';
// import GameBoard from './GameBoard.js'
// import UserBoards from './UserBoards.js';
// import Login from './Login.js';

// import { ThemeProvider as Theme, defaultTheme} from '@material-ui/core/styles';

// import { BrowserRouter as Router } from 'react-router-dom';
// import { createRender } from '@material-ui/core/test-utils';
// import Adapter from 'enzyme-adapter-react-16';
// import Enzyme from 'enzyme'

// Enzyme.configure({ adapter: new Adapter() })

// export const withMaterial = (Component, props) => {
//   const render = createRender();
//   return render(
//       <Router>
//         <Theme theme={defaultTheme}>
//           <Component {...props} />
//         </Theme>
//       </Router>
//     );
// };

// // test.only('renders App.js correctly', () => {
// //   const MaterialApp = withMaterial(App)
// //   const tree = renderer
// //   .create(<MaterialApp />)
// //   .toJSON();
// //   expect(tree).toMatchSnapshot();
// // });

// test.only('renders Landing.js correctly', () => {
//   const MaterialApp = withMaterial(Landing)
//   const tree = renderer
//   .create(<MaterialApp />)
//   .toJSON();
//   expect(tree).toMatchSnapshot();
// });

// test('renders GameBoard.js correctly', () => {
//   const tree = renderer
//   .create(<GameBoard />)
//   .toJSON();
//   expect(tree).toMatchSnapshot();
// });
// test('renders Login.js correctly', () => {
//   const tree = renderer
//   .create(<Login />)
//   .toJSON();
//   expect(tree).toMatchSnapshot();
// });
// test('renders UserBoards.js correctly', () => {
//   const tree = renderer
//   .create(<UserBoards />)
//   .toJSON();
//   expect(tree).toMatchSnapshot();
// });