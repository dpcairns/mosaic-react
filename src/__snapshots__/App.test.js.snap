// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders App.js correctly 1`] = `
<div>
  <h1>
    UserInfos
  </h1>
  <div>
    <h1>
      Mosaic
    </h1>
    <button
      className="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary"
      disabled={false}
      onBlur={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex={0}
      type="button"
    >
      <span
        className="MuiButton-label"
      >
        <span
          className="MuiButton-startIcon MuiButton-iconSizeMedium"
        >
          <svg
            aria-hidden="true"
            className="MuiSvgIcon-root"
            focusable="false"
            role="presentation"
            viewBox="0 0 24 24"
          >
            <path
              d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm-2 14.5v-9l6 4.5-6 4.5z"
            />
          </svg>
        </span>
        Play
      </span>
    </button>
    <button
      className="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedSecondary"
      disabled={false}
      onBlur={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex={0}
      type="button"
    >
      <span
        className="MuiButton-label"
      >
        <span
          className="MuiButton-startIcon MuiButton-iconSizeMedium"
        >
          <svg
            aria-hidden="true"
            className="MuiSvgIcon-root"
            focusable="false"
            role="presentation"
            viewBox="0 0 24 24"
          >
            <path
              d="M12 3c-4.97 0-9 4.03-9 9s4.03 9 9 9c.83 0 1.5-.67 1.5-1.5 0-.39-.15-.74-.39-1.01-.23-.26-.38-.61-.38-.99 0-.83.67-1.5 1.5-1.5H16c2.76 0 5-2.24 5-5 0-4.42-4.03-8-9-8zm-5.5 9c-.83 0-1.5-.67-1.5-1.5S5.67 9 6.5 9 8 9.67 8 10.5 7.33 12 6.5 12zm3-4C8.67 8 8 7.33 8 6.5S8.67 5 9.5 5s1.5.67 1.5 1.5S10.33 8 9.5 8zm5 0c-.83 0-1.5-.67-1.5-1.5S13.67 5 14.5 5s1.5.67 1.5 1.5S15.33 8 14.5 8zm3 4c-.83 0-1.5-.67-1.5-1.5S16.67 9 17.5 9s1.5.67 1.5 1.5-.67 1.5-1.5 1.5z"
            />
          </svg>
        </span>
        Color Switch
      </span>
    </button>
  </div>
</div>
`;
